src/server/contentSecurityPolicy.ts

```diff
@@ -228,7 +229,14 @@ const contentSecurityPolicy = {
     upgradeInsecureRequests: config.runtimeType === "development" || config.ndlaEnvironment === "local" ? null : [],
     scriptSrc,
     frameSrc,
-    frameAncestors: null,
+    frameAncestors: [
+      (req: IncomingMessage, _: ServerResponse) => {
+        if (req.url?.includes("-iframe") || req.url?.includes("/lti")) {
```

Comment:
Kan man ikke lure seg rundt frame-ancestors på denne måten?

`curl -I http://localhost:3000/subject:1:ab5e9191-407a-492e-a2b9-5071275a37a7/topic:3:183731/topic:4ecd79b5-3b02-4186-89c9-c834e3878b4a/resource:1:4033/lti`

så får man jo `frame-ancestors *;`?

Reply:
Sant. Kan sjekke om path begynner på /lti eller /lang/lti i stedet for.

Reply:
Kan fremdeles lure seg rundt med `-iframe` da, men jeg vet ikke om det er så farlig om folk klarer å embedde ndla på hvilken som helst side jeg?

Reply:
Tenker dette er ganske liten risk